packages/cli/src/commands/ui/create/angular.spec.ts

```diff
@@ -116,13 +128,26 @@ describe('ui:create:angular', () => {
     doMockPreconditions();
     preconditionStatus.npm = true;
     preconditionStatus.node = true;
+    preconditionStatus.apiKey = true;
   });
 
   afterEach(() => {
     mockedIsNodeVersionInRange.mockClear();
     mockedIsNpmVersionInRange.mockClear();
   });
 
+  test
```

Comment:
That's come from a nice sentiment, but I think it wont be necessary, if you check for the current preconditions, we check either if one is wrong or if all are OK.

I don't think you want or need more necessarly, you're better doing rigorous UT on the precondition function itself ;)